### PR TITLE
[MULTIARCH3470] Remove KVM only restriction

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.14+

Issue: https://issues.redhat.com/browse/MULTIARCH-3470

Link to docs preview:
- [Installing an on-premise cluster using the Assisted Installer](https://67579--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_on_prem_assisted/installing-on-prem-assisted)
- [Choosing a method to install OpenShift Container Platform on IBM Z or IBM® LinuxONE](https://67579--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/preparing-to-install-on-ibm-z#choosing-an-method-to-install-ocp-on-ibm-z)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by holgwolf** [comment]
LGTM

File: installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc [hunk 1]
```diff
@@ -6,12 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can install {product-title} on on-premise hardware or on-premise VMs using the {ai-full}. Installing {product-title} using the {ai-full} supports x86_64, AArch64, ppc64le, and s390x CPU architectures.
-
-[NOTE]
-====
-Installing {product-title} on {ibmzProductName} (s390x) is supported only with RHEL KVM installations.
-====
+You can install {product-title} on on-premise hardware or on-premise VMs using the {ai-full}. Installing {product-title} using the {ai-full} supports `x86_64`, `AArch64`, `ppc64le`, and `s390x` CPU architectures.
```
Comment (tmalove) [inline installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc:9]:
```suggestion
You can install {product-title} on on-premise hardware or on-premise VMs using the {ai-full}. Installing {product-title} by using the {ai-full} supports `x86_64`, `AArch64`, `ppc64le`, and `s390x` CPU architectures.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u

**Comment by adellape** [comment]
DM'd question for more details re: whether QE or change-management are needed, considering it's changing a support statement.

**Comment by adellape** [comment]
Discussed offline that an announce-only change management email was sufficient (per DPM). Merging!

**Pull request merged**